R/g.calibrate.R

```diff
@@ -141,12 +141,7 @@ g.calibrate = function(datafile, params_rawdata = c(),
           data = P$data
         }
       } else if (dformat == FORMAT$AD_HOC_CSV) {
-        if (length(params_rawdata[["rmc.col.time"]]) > 0 && mon == MONITOR$AD_HOC) {
-          columns_to_use = 2:4
-        } else {
-          columns_to_use = 1:3
-        }
-        data = P$data[, columns_to_use]
+        data = P$data[, c(params_rawdata[["rmc.col.acc"]],  params_rawdata[["rmc.col.temp"]])]
```

Comment:
I think this might actually be incorrect?

We get `P$data` from `read.myacc.csv`, and there it gets populated with columns c("timestamp","accx","accy","accz","temperature"), in that exact order. You can see it here:
https://github.com/wadpac/GGIR/blob/master/R/read.myacc.csv.R#L172-L179

So it doesn't matter where the acceleration and temperature columns were located in the original csv file. `P$data$temperature` is always 4th or 5th column (depending on whether there is a timestamp or not), no matter what `params_rawdata[["rmc.col.temp"]]` value was. And acceleration columns are always either 1:3 or 2:4, no matter what `params_rawdata[["rmc.col.acc"]]` values were.

I'll send an alternative PR in a bit.

Reply:
Thanks for spotting that Lena, I will now close this PR.